src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Filter panel
+description: The filter panel is a component that can be customized to provide categorized filters to help users trim down large amounts of content, data and/or search results.
+---
+
+import { ComponentStatus } from 'components/ComponentList';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentFeedback from 'components/ComponentFeedback';
+
+<ComponentDescription name="Filter panel" type="ui" />
```

Comment:
Looks like you need to add `Filter panel` [over here](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/main/src/data/components.json#L3) under `ui`